cluster/ubuntu/download-release.sh

```diff
@@ -52,8 +52,21 @@ grep -q "^${ETCD_VERSION}\$" binaries/.etcd 2>/dev/null || {
   echo ${ETCD_VERSION} > binaries/.etcd
 }
 
+function get_latest_version_number {
+  local -r latest_url="https://storage.googleapis.com/kubernetes-release/release/stable.txt"
+  if [[ $(which wget) ]]; then
+    wget -qO- ${latest_url}
+  elif [[ $(which curl) ]]; then
+    curl -Ss ${latest_url}
+  else
+    echo "Couldn't find curl or wget.  Bailing out."
+    exit 4
+  fi
+}
+
+KUBE_VERSION=`echo $(get_latest_version_number) | sed 's/v//'`
```

Comment:
> Since you suggested these changes, I'd also try to push them to the shell
> script that resides below, on which my change was originally based upon.

That would be appreciated :)